conf.toml

```diff
@@ -2,6 +2,6 @@
 "DbPassword" = "changeme"
 "DbName" = "app"
 "DbPort" = "5432"
-"DbHost" = "localhost"
-"ListenHost" = "localhost"
+"DbHost" = "localhost3"
+"ListenHost" = "0.0.0.0"
```

Comment:
Listner Host updated to 0.0.0.0 to fix the issue with docker web application working at host level